src/js/stories/DropdownStories.jsx

```diff
@@ -13,48 +15,114 @@ const style = {
     flexDirection: 'column',
     justifyContent: 'space-around',
     alignItems: 'center',
-    height: 400,
     padding: 10,
 };
 
-function createOptions(count) {
+const a2z = 'ABCDEFGHIJKLM NOPQRSTUVWXYZ';
+
+function createOptions(text = 'Option', asObject = false) {
     const options = [];
-    options.push('ABCDEFGHIJKLMNOPQRSTUVWXYZ');
-    for (let index = 0; index < count; index++) {
-        options.push(`Option ${options.length + 1}`);
+
+    for (let index = 0; index < 200; index++) {
+        const label = `${text} ${options.length + 1}`;
+        options.push(!asObject ? label : { label });
     }
     return options;
 }
 
+
+
 function Default() {
+    Utils.createCssRule(
```

Comment:
You're right, it can create some dup style rules and be a bit funky. I could defend against that within the method? I guess my thought was for storybook it was ok to do things a little hacky, but I don't mind trying to clean it up before we merge.